epictrack-web/src/components/work/workStaff/WorkStaffList.tsx

```diff
@@ -70,11 +75,26 @@ const WorkStaffList = () => {
               .filter((p: { role: { name: any } }) => p.role.name === rolename)
               .map(
                 (p: { first_name: string; last_name: string }) =>
-                  `${p.first_name} ${p.last_name}`
+                  `${p.last_name} ${p.first_name}`
               )
-              .join(", ")}`,
+              .join("; ")}`,
           enableHiding: false,
           enableColumnFilter: true,
+          Cell: ({ row, renderedCellValue }) => {
+            const staff = row.original.staff
+              .filter((p: { role: { name: any } }) => p.role.name === rolename)

```

Comment:
please don't use single letter variables

Reply:
changed all the variables should be more readable now